ecosystem/sep-0012.md

```diff
@@ -119,21 +120,21 @@ GET [KYC_SERVER]/customer?account=<muxed-account>&type=<customer-type>
+| ---------------- | --------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `id`             | `string`                          | (optional) The ID of the customer as returned in the response of a previous `PUT` request. If the customer has not been registered, they do not yet have an `id`.                                                                                                                                                                                                                                                                                                                          |
+| `account`        | `G...` or `M...` or `C...` string | (**deprecated**, optional) The server should infer the account from the `sub` value in the SEP-10/SEP-45 JWT to identify the customer. The `account` parameter is only used for backwards compatibility, and if explicitly provided in the request body it should match the `sub` value of the decoded SEP-10/SEP-45 JWT.                                                                                                                                                                  |
+| `memo`           | string                            | (optional) the client-generated [memo](https://developers.stellar.org/docs/glossary/transactions/#memo) that uniquely identifies the customer. If a memo is present in the decoded SEP-10 JWT's `sub` value, it must match this parameter value. If a muxed account is used as the JWT's `sub` value, memos sent in requests must match the 64-bit integer subaccount ID of the muxed account. See the [Shared Accounts](#shared-omnibus-or-pooled-accounts) section for more information. |
+| `memo_type`      | string                            | (**deprecated**, optional) type of `memo`. One of `text`, `id` or `hash`. Deprecated because memos should always be of type `id`, although anchors should continue to support this parameter for outdated clients. If `hash`, `memo` should be base64-encoded. If a memo is present in the decoded SEP-10 JWT's `sub` value, this parameter can be ignored. See the [Shared Accounts](#shared-omnibus-or-pooled-accounts) section for more information.                                    |
```

Comment:
```suggestion
| `memo_type`      | string                            | (**deprecated**, optional) type of `memo`. One of `text`, `id` or `hash`. Deprecated because memos should always be of type `id`, although anchors should continue to support this parameter for outdated clients. If `hash`, `memo` should be base64-encoded. If a memo is present in the decoded SEP-10 JWT's `sub` value, this parameter can be ignored. If the `account` is a `C...` account, the `memo_type` must not be specified. See the [Shared Accounts](#shared-omnibus-or-pooled-accounts) section for more information.                                    |
```